src/test/shared/clients/defaultEc2Client.test.ts

```diff
@@ -93,6 +93,97 @@ describe('extractInstancesFromReservations', function () {
                 actualResult
             )
         })
+
+    it('is able to process results without complete Tag field.', async function () {
```

Comment:
"should", "is able", "correctly" are usually redundant in test descriptions.
 
```suggestion
    it('processes results without complete Tag field.', async function () {
```